command_uninstall.go

```diff
@@ -37,6 +37,18 @@ func cmdUninstall(c *cli.Context) error {
 }
 
 func uninstallPackage(cmd string) error {
+	Outer:
```

Comment:
This label is not that descriptive, is there something better we can give this? (or as mentioned in a previous comment, switch to a `goto`)